### DISCUSSION TITLE
The teacher benefit application cannot be processed after uploading the academic proof.

### DISCUSSION THREAD

**Discussion by GeoSpatialX** [community/community#35303] [GitHub Education]
I have fixed the following items: Thanks.

![image](https://user-images.githubusercontent.com/115056707/194235468-f191f4a9-7905-4357-860a-a0eb97216214.png)

**Comment by samus-aran** [comment]
Hi @GeoSpatialX  👋
We'd love to help! Thanks for the screenshot. You said you've corrected these actions so are there any new rejection reasons since correcting them. This might help someone give you a nudge in the right direction.

**Reply by GeoSpatialX** [reply]
I think I have addressed the above three, but still the same warnings. For the first one, I open the setting of chrome browser and pick the option of "sites can ask for your location". Is that correct? How could specify Github to use my location information?
